<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Papillon</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../papillon.qrc">
    <normaloff>:/images/main.png</normaloff>:/images/main.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Profiles"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>About(&amp;A)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuLanguage_L">
    <property name="title">
     <string>Language(&amp;L)</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_About"/>
   <addaction name="menu_Help"/>
   <addaction name="menuLanguage_L"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="actionHelp"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normalon>:/images/document-new.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>new</string>
   </property>
   <property name="toolTip">
    <string>new a  code file</string>
   </property>
   <property name="statusTip">
    <string>new a  code file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>:/images/document-open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>open a code file</string>
   </property>
   <property name="statusTip">
    <string>open a code file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/images/document-save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>save the code file</string>
   </property>
   <property name="statusTip">
    <string>save the code file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset>
     <normalon>:/images/document-save-as.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>save as</string>
   </property>
   <property name="toolTip">
    <string>save the code file as another file</string>
   </property>
   <property name="statusTip">
    <string>save the code file as another file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normalon>:/images/application-exit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
   <property name="toolTip">
    <string>quit Papillon</string>
   </property>
   <property name="statusTip">
    <string>quit Papillon</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normalon>:/images/edit-copy.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
   <property name="toolTip">
    <string>copy some code</string>
   </property>
   <property name="statusTip">
    <string>copy some code</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normalon>:/images/edit-cut.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>cut</string>
   </property>
   <property name="toolTip">
    <string>cut some code</string>
   </property>
   <property name="statusTip">
    <string>cut some code</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normalon>:/images/edit-paste.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>paste</string>
   </property>
   <property name="toolTip">
    <string>paste some code</string>
   </property>
   <property name="statusTip">
    <string>paste some code</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normalon>:/images/edit-undo.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
   <property name="toolTip">
    <string>undo some operation</string>
   </property>
   <property name="statusTip">
    <string>undo some operation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normalon>:/images/edit-redo.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
   <property name="toolTip">
    <string>redo some operation</string>
   </property>
   <property name="statusTip">
    <string>redo some operation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normalon>:/images/about.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>about</string>
   </property>
   <property name="toolTip">
    <string>about of Papillon</string>
   </property>
   <property name="statusTip">
    <string>about of Papillon</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normalon>:/images/help.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>help</string>
   </property>
   <property name="toolTip">
    <string>help for Papillon</string>
   </property>
   <property name="statusTip">
    <string>help for Papillon</string>
   </property>
  </action>
  <action name="actionOpen_Profiles">
   <property name="icon">
    <iconset>
     <normalon>:/images/profiles-open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>open profiles</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../papillon.qrc"/>
 </resources>
 <connections/>
</ui>
